images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -49,6 +49,12 @@ function Get-NodeVersion {
     return "Node $nodeVersion"
 }
 
+function Get-PerlVersion {
+    $result = Get-CommandResult "perl --version | perl -ne 'print /(\d\.\d+\.\d+)/'"
```

Comment:
nitpick: `perl -e 'print substr($^V, 1)'`

Reply:
So true...

Reply:
Um, having trouble with the $ inside a PS command...

Reply:
I've used that, but it didn't work for some reason (probably related to the $ thingie) inside `Get-CommandResult` I can go back to that version if calling it is mandatory.